test/multi-runtime-events.test.ts

```diff
@@ -207,22 +198,15 @@ describe('Multi Runtime', () => {
                         await delayPromise(1500);
                     }
 
-                    test();
+                    test().catch(() => cleanOpenRuntimes());
                 });
 
                 it('should raise application-started events', function (done: (value: void) => void) {
```

Comment:
event name is better to change to 'application-created' to match below event handling codes.